app/controllers/bgeigie_imports_controller.rb

```diff
@@ -95,6 +95,15 @@ def destroy
     end
   end
 
+  def kml
+    bgeigie_import = BgeigieImport.find(params[:id])
+    ::Actions::BgeigieImports::Kml.new(
```

Comment:
I was gonna use `render`, but it didn't work maybe because of extension (.kml). I will try again once we update to rails 4. I will add comment about it later.

For `Actions` concept, 2 reasons. 1) Mentioned above 2) ABC size.
 If we can do like `render 'beigie_logs', formats: [:kml], content_type: Mime::KML`, we can get rid of this class. Although, I like concept of "hexagonal architecture" or "use case" and would like to extract some of actions into classes for reduce class length and/or method complexity metrics.